360/watchapp/templates/watchapp/homepage.html

```diff
@@ -1013,28 +1038,6 @@ <h2>{{ friend.username }}</h2>
             localStorage.setItem('isOnline', isOnline.checked.toString());
         });
     </script>
-    
-    <script>
-        // script to generate notifications dropdown menu generated from copilot
-
-        $(document).ready(function() {
-            $('#notifications-button').click(function() {
```

Comment:
Or any of this?

Reply:
moved it up and added the settings dropdown with it